aws/resource_aws_spot_fleet_request.go

```diff
@@ -362,6 +455,28 @@ func buildSpotFleetLaunchSpecification(d map[string]interface{}, meta interface{
 		opts.WeightedCapacity = aws.Float64(wc)
 	}
 
+	if v, ok := d["tag"]; ok && v != "" {
+		tags, _ := spotFleetTagsFromMap(
+			setToMapByKey(v.(*schema.Set), "key"))
+
+		opts.TagSpecifications = []*ec2.SpotFleetTagSpecification{
+			{
+				ResourceType: aws.String(ec2.ResourceTypeInstance),
+				Tags:         tags,
+			},
+		}
+	}
+
+	if v, ok := d["tags"]; ok && v != "" {
+		tags, _ := spotFleetTagsFromList(v.([]interface{}))
+		opts.TagSpecifications = []*ec2.SpotFleetTagSpecification{
+			{
+				ResourceType: aws.String(ec2.ResourceTypeInstance),
+				Tags:         tags,
+			},
+		}
+	}
```

Comment:
This was originally in the reference implementation, but caused some minor issues that I didn't have capacity to work on at the time. I intend to add it back in.